src/chatlist/chatList.js

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import { withStyles } from "@material-ui/core/styles";
+import List from "@material-ui/core/List";
+import ListItem from "@material-ui/core/ListItem";
+import ListItemText from "@material-ui/core/ListItemText";
+import ListItemAvatar from "@material-ui/core/ListItemAvatar";
+import Avatar from "@material-ui/core/Avatar";
+import Typography from "@material-ui/core/Typography";
+import styles from "./styles";
+import Divider from "@material-ui/core/Divider";
+import Button from "@material-ui/core/Button";
+import ListItemIcon from "@material-ui/core/ListItemIcon";
+import NotificationImportant from "@material-ui/icons/NotificationImportant";
+
+class ChatListComponent extends React.Component {
+  newChat = () => {
+    console.log("New chat click");
+  };
+
+  selectChat = (index) => {
+    console.log("select chat", index);
+  };
+  render() {
+    const { classes } = this.props;
+
+    if (this.props.chats.length > 0) {
```

Comment:
While this is good that it's explicit, it's worth knowing that `0` is a falsy value. You could just make this line `if (this.props.chats.length) {` and it'd work exactly the same.

Reply:
Yep, I corrected the line.